share/completions/equery.fish

```diff
@@ -1,104 +1,123 @@
+complete -c equery -n '__fish_seen_subcommand_from list' -s p -l portage-tree      -d "list packages in the main portage tree"
+#complete -c equery -n '__fish_seen_subcommand_from list' -s F -l format=TMPL       -d "specify a custom output format"
+
+# meta
+complete -c equery -n '__fish_seen_subcommand_from meta' -s d -l description       -d "show an extended package description"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s H -l herd              -d "show the herd(s) for the package"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s k -l keywords          -d "show keywords for all matching package versions"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s l -l license           -d "show licenses for the best maching version"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s m -l maintainer        -d "show the maintainer(s) for the package"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s S -l stablreq          -d "show STABLEREQ arches (cc's) for all matching package versions"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s u -l useflags          -d "show per-package USE flag descriptions"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s U -l upstream          -d "show package's upstream information"
+complete -c equery -n '__fish_seen_subcommand_from meta' -s x -l xml               -d "show the plain metadata.xml file"
 
 # size
-complete -c equery -n '__fish_seen_subcommand_from size' -s b -l bytes -d "report size in bytes"
+complete -c equery -n '__fish_seen_subcommand_from size' -s b -l bytes             -d "report size in bytes"
+complete -c equery -n '__fish_seen_subcommand_from size' -s f -l full-regex        -d "query is a regular expression"
 
 # uses
-complete -c equery -n '__fish_seen_subcommand_from uses' -s a -l all -d "include non-installed packages"
-# END Local opts
-#########################################################################
-
-#########################################################################
-# Arguments
-# Commands applied to a package which doesn't need to be installed
-complete -c equery -n '__fish_seen_subcommand_from depends depgraph hasuse which' -xa '(__fish_equery_print_all_pkgs)'
-
-# Commands applied to a package which has to be installed
-complete -c equery -n '__fish_seen_subcommand_from check files uses size' -xa '(__fish_equery_print_installed_pkgs)'
-# END Arguments
-#########################################################################
\ No newline at end of file
+complete -c equery -n '__fish_seen_subcommand_from uses' -s a -l all               -d "include all package versions"
+complete -c equery -n '__fish_seen_subcommand_from uses' -s i -l ignore-l10n       -d "don't show l10n USE flags"
+
+# which
+complete -c equery -n '__fish_seen_subcommand_from which' -s m -l include-masked    -d "return highest version ebuild available"
```

Comment:
Is that actually the definition?

Can't I have a masked ebuild with a lower version number? From my combined four hours of using Gentoo, I recall seeing ebuilds with a version number of "9999" or something like that.

Reply:
The definition is copypasted from `equery which -h`.
The man-page is a only little more elaborate: "Return the path to the hightest version ebuild available."